lib/oli_web/templates/project/overview.html.eex

```diff
@@ -115,6 +118,61 @@
           <% end %>
         </div>
       </div>
+
+      <div class="row my-4">
+        <div class="col-12">
+          <h4>Delete this course</h4>
+          <button type="button" class="btn btn-danger" onclick="showDeleteModal()">Delete this Course</button>
+        </div>
+      </div>
+
     </div>
   </div>
 </div>
+<script>
+  const showDeleteModal = () => {
+    $('#delete-modal').modal('show');
+  }
+  $(function () {
+    const deleteTitleInput = document.querySelector('#delete-confirm-title');
+    if (deleteTitleInput) {
+      deleteTitleInput.addEventListener("input", function (e) {
+        const value = e.target.value;
+        console.log(value)
+        const deleteSubmitButton = document.querySelector('#delete-modal-submit')
+        if (value === '<%= @project.title %>') {
+          deleteSubmitButton.disabled = false;
+        } else {
+          deleteSubmitButton.disabled = true;
+        }
+      });
+    }
+  });
+</script>
+<div class="modal fade" id="delete-modal" tabindex="-1" role="dialog" aria-labelledby="delete-modal" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">Are you absolutely sure?</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <div class="container form-container">
+          <div><p>Please type <strong><%= @project.title %></strong> below to confirm.</p></div>
+              <%= form_for @conn, Routes.project_path(@conn, :delete, @project), fn f -> %>
```

Comment:
Expand the messaging here to include an explanation that any existing course sections that are using this project will continue to operate.

Reply:
Expanded message as suggested